bin/start-server.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+let server = require('../lib/server')
+let port = process.env.PORT || 8080
```

Comment:
Was spricht hier gegen `const` an stelle von `let`?

Gleiches auch in `/lib/server.js`